assets/css/components/search.css

```diff
@@ -14,6 +14,10 @@
     }
   }
 
+  .hextra-search-input:focus {
+    @apply hx:hextra-focus
+  }
```

Comment:
should we add the `hx:focus:hextra-focus` inline to the search input directly?

Reply:
yes, as it's a utility, it can be better.